.eslintrc.json

```diff
@@ -0,0 +1,117 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "standard-react"
+  ],
+  "env": {
+    "browser": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
+    "jest/no-disabled-tests": "error",
+    "jest/no-focused-tests": "error",
+    "jest/no-identical-title": "error",
+    "jest/prefer-to-have-length": "error",
+    "jest/prefer-to-be-null": "error",
+    "jest/valid-expect": "error",
+    "jest/valid-describe": "error",
+    "jest/no-test-prefixes": "error",
+    "jest/consistent-test-it": "error",
+    "react/jsx-uses-react": "error",
+    "react/jsx-uses-vars": "error",
+    "react/jsx-indent": "off",
+    "react/jsx-indent-props": "off",
+    "react/prop-types": "error",
+    "react/jsx-tag-spacing": "error",
+    "react/no-unused-prop-types": "error",
+    "react/no-access-state-in-setstate": "error",
+    "react/prefer-es6-class": [
+      "error",
+      "always"
+    ],
+    "react/no-deprecated": "error",
+    "react/no-did-mount-set-state": "error",
+    "react/no-did-update-set-state": "error",
+    "react/no-direct-mutation-state": "error",
+    "react/no-typos": "error",
+    "sort-imports-es6-autofix/sort-imports-es6": [
```

Comment:
how does this sort the imports? is it turned off? because I don't like how the imports are sorted in the RouterFragment

Reply:
Default sorting is kind of shit. Its currectly off. If you want you can enable it and see for yourself. It only can sort alphabetically.

Reply:
meeeh :/